website/pages/docs/advanced-topics/monitoring/otel-datadog.md

```diff
@@ -1,66 +1,15 @@
 ---
-title: Monitoring CloudQuery
-description: Short walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs
+title: OpenTelemetry and Datadog
+description: Walkthrough on how to use OpenTelemetry to monitor CloudQuery Syncs with Datadog
 ---
 
-# Monitoring
-
-Monitoring CloudQuery can be done in a number of main ways:
-
-- Logging
-- OpenTelemetry
-
-## Logging
-
-CloudQuery utilizes structured [logging](../reference/cli/cloudquery) (in plain and JSON formats) which can be analyzed by local tools such as `jq`, `grep` and remote aggregations tools like `loki`, `datadog` or any other popular log aggregation that supports structured logging.
-
-## OpenTelemetry (Preview)
-
-ELT workloads can be long running and sometimes it is necessary to better understand what calls are taking the most time; to potentially optimize those on the plugin side, ignore them or split them to a different workload. Plugins come with an OpenTelemetry library built in, but it is up to the plugin author to instrument the most important parts--usually the API calls--this way it is possible to see what calls take the longest time, or where throttling and errors are happening.
```

Comment:
Cleaned up this sentence a bit since plugin authors don't really need to do anything most of the time